ush/forecast_postdet.sh

```diff
@@ -438,6 +438,10 @@ MOM6_postdet() {
 
       # Daily output
       if (( fhr > 0 & fhr % 24 == 0 )); then
+	   
+	# For daily output, the valid date in the file is based on the beginning of avg. period
+        vdate=$(date --utc -d "${vdate:0:8} ${vdate:8:2} - 24 hours" +%Y%m%d%H)
```

Comment:
Did the patch from @aerorahul misalign the indentation blocks?

Reply:
There were some tabs I unintentionally put in.  Hopefully this is fixed now.